src/main/java/knightminer/inspirations/building/BuildingClientProxy.java

```diff
@@ -173,10 +185,24 @@ public void onModelBake(ModelBakeEvent event) {
 		replaceBookshelfModel(event, InspirationsBuilding.shelf_rainbow);
 		replaceBookshelfModel(event, InspirationsBuilding.shelf_tomes);
 
-		if(InspirationsBuilding.enlightenedBush != null) {
-			ResourceLocation location = InspirationsBuilding.enlightenedBush.getRegistryName();
-			replaceTexturedModel(event, new ModelResourceLocation(location, ""), "leaves", true);
-		}
+		replaceBushModel(event, InspirationsBuilding.whiteEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.redEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.blueEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.greenEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.rainbowEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.christmasEnlightenedBush);
+	}
+
+	private static void replaceBushModel(ModelBakeEvent event, BlockEnlightenedBush bush) {
+		ResourceLocation bushLoc = Objects.requireNonNull(bush.getRegistryName());
```

Comment:
This function seems pretty generic, any reason it is not just pulled into a variant of replaceTextureModel?

Reply:
Not really no, I should do so.